packages/block-editor/src/components/block-list/use-block-props/index.js

```diff
@@ -128,6 +128,11 @@ export function useBlockProps( props = {}, { __unstableIsHtml } = {} ) {
 		} ),
 	] );
 
+	const updatedStyle = { ...wrapperProps.style, ...props.style };
+	const style = useMemo( () => updatedStyle, [
+		JSON.stringify( updatedStyle ),
+	] );
```

Comment:
It feels like a code smell to me as `useMemo` isn't designed to be use this way, and I also doubt that it would really make a difference?

Reply:
Sure, let's drop this in this PR, though I do want to revisit `useBlockProps` and other hooks returning a new style object each time. It's pops up as a render reason that `hooks changed` and inspecting deeper we often see that the style object triggered it, sometimes with an empty object.

For context, the style object appeared simple enough to try JSON.stringify

https://twitter.com/dan_abramov/status/1104414272753487872

Reply:
Thanks! Dropped in 9066a9b